core/db.py

```
@@ -106,9 +106,12 @@ def execute(self, statement: Any, *args: Any, **kwargs: Any):
             result = conn.execute(stmt, *args, **kwargs)
             try:
                 conn.commit()
-            except Exception:
+            except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)